packages/portal/src/app/shared/listing.service.ts

```diff
@@ -80,6 +83,13 @@ export class ListingService {
     if (resource.status !== 200) {
       throw new Error(checkoutSession.error);
     }
+    const listing = await this.getListingById(reservationDetails.listingId ?? "");
+    if (listing) {
+      this.realtimeService.broadcastCheckoutNotification(listing, reservationDetails.from, reservationDetails.to);
```

Comment:
I haven't gotten yet to the implementation of this broadcast. Maybe we can consolidate to one?